docs/usage/network_policies.md

```diff
@@ -0,0 +1,148 @@
+
+The policy `allow-to-blocked-cidrs` allows pods with the label `networking.gardener.cloud/to-blocked-cidrs=allowed` to access IPs that are explicitly blocked for all control planes in a `Seed`cluster (configurable via `seed.Spec.networks.blockCIDRS`). This is used for instance to block the cloud provider's metadata service.
+
+Another network policy to be highlighted is `allow-to-seed-apiserver`.
+Some components need access to the `Seed` API Server. This can be allowed by labeling the pod with `networking.gardener.cloud/to-seed-apiserver=allowed`.
+This policy allows exactly the IPs of the `kube-apiserver` of the `Seed`.
+While all other policies have a static set of permissions (do not change during the lifecycle of the `Shoot`), the policy `allow-to-seed-apiserver` is reconciled to reflect the endpoints in the `default` namespace.
+This is required because endpoint IPs are not stable (think of scaling the `Seed` API Server pods or hibernating the `Seed` cluster in a local development environment).
+
+Further the following network policies exist in the `Shoot` namespace.
+These policies are the same for every `Shoot` control plane.
+
+```
+NAME                              POD-SELECTOR      
+# Pods that need to access the Shoot API server. Used by all Kubernetes control plane components.
+allow-to-shoot-apiserver          networking.gardener.cloud/to-shoot-apiserver=allowed 
+
+# allows access to kube-dns/core-dns pods for DNS queries                       
+allow-to-dns                      networking.gardener.cloud/to-dns=allowed
+
+# allows access to private IP address ranges 
+allow-to-private-networks         networking.gardener.cloud/to-private-networks=allowed  
+
+# allows access to all but private IP address ranges 
+allow-to-public-networks          networking.gardener.cloud/to-public-networks=allowed                     
+
+# allows Ingress to etcd pods from the Shoot's Kubernetes API Server
+allow-etcd                        app=etcd-statefulset,garden.sapcloud.io/role=controlplane   
+
+# used by the Shoot API server to allows ingress from pods labeled
+# with'networking.gardener.cloud/to-shoot-apiserver=allowed', from Prometheus, and allows Egress to etcd pods
+allow-kube-apiserver              app=kubernetes,garden.sapcloud.io/role=controlplane,role=apiserver       
+```
+
+
+### Network policies for Logging & Monitoring 
+
+[Gardener provides out-of-the-box logging and monitoring](https://github.com/gardener/gardener/blob/master/docs/extensions/logging-and-monitoring.md) for the `ControlPlane` via a stack based on prometheus, elasticsearch, kibana and grafana.
+
+{{% notice tip %}}
+<p>Gardener currently introduces a logging stack based on [Loki](https://github.com/grafana/loki). So this section is subject to change. Please checkout [the Comunity Meeting for more information](https://www.youtube.com/watch?v=345b8xCcB-U&t=1166s).</p>
+{{% /notice %}}
```

Comment:
Which kind of syntax is this? It's not properly rendered in the markdown: https://github.com/gardener/gardener/blob/d90cb6f1646d3078be69a4883770db3ff69d451f/docs/usage/network_policies.md